classes/Player.py

```diff
@@ -39,11 +39,10 @@ def bullet_shoot(self,bullet_group):
     def hit_target(self,target_group,display_surface):
         if pygame.sprite.spritecollide(self,target_group,False):
             self.game_over=True
-            game_over_text = f'GameOver!!'
-            
-            while self.game_over:
-                text_surf = self.font.render(game_over_text,True,(255,255,255))
-                text_rect = text_surf.get_rect(center = (window_width / 2, window_height /2))
+            game_over_text = f'Game Over!!'
+            text_surf = self.font.render(game_over_text,True,(255,255,255))
+            text_rect = text_surf.get_rect(center = (window_width / 2, window_height /2))
+            while self.game_over:                
```

Comment:
Great to see these changes are made and the code is working.
These new updated changes look good to me. Will merge it.